src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
You don't need to pass health to a constructor. Just set it to 100.
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```